modules/generator/instance.go

```diff
@@ -108,6 +108,11 @@ func newInstance(cfg *Config, instanceID string, overrides metricsGeneratorOverr
 	}
 	go i.watchOverrides()
 
+	// add ingestion slack overrides
+	if slack := overrides.MetricsGeneratorIngestionSlack(instanceID); slack > 0 {
```

Comment:
It might make sense to move this to the `updateProcessors` so that if it changes, we don't have to restart the instance for changes to take effect.

Reply:
good call. overrides should be dynamically loaded.